### ISSUE TITLE
Covering other jq locations

### ISSUE THREAD

**Issue by chrisgrieser** [gohoyer/Alfred-Cheat.sh#2]
thanks for the nice workflow!

I tried using it, however I got an error with `jq`. You can fix that by adding the following line at the top:

```sh
export PATH=/usr/local/bin/:/opt/homebrew/bin/:$PATH
```

on M1 Macs, HomeBrew installs `jq` at `/opt/homebrew/bin/`, and adding that to the PATH ensures that you can use `jq` like in the shell, without worrying where a user has installed it.

```sh
#!/bin/zsh
export PATH=/usr/local/bin/:/opt/homebrew/bin/:$PATH

{{...}}

json_array=$(printf '%s
' "${cheat_list[@]}" | grep -v ":\|/" | jq -R . | jq -s .)

# Format the output as Script Filter Json Format
# shellcheck disable=SC2016
jq -n --argjson item "$json_array" -f <(echo '{"items":[$item[] as $name | {"uid":$name,"title":$name,"arg":$name,"autocomplete":$name}]}')

```

**Comment by gohoyer** [comment]
Thanks for reporting this issue and pointing the fix! I'll put this on the next release soon!

**Comment by chrisgrieser** [comment]
just a minor thing since it seems you also develop small Alfred workflow:

Change this to `external script` to save the file seperately from the `info.plist`. That enables other's to easier see the underlying code and make PRs (e.g. I could have made a PR for this issue to save you some time :) )

<img width="333" alt="Screenshot 2022-02-14 16 39 22" src="https://user-images.githubusercontent.com/73286100/153895760-e57b16f3-d064-4a0b-826b-99bd8b9112ac.png">

**Comment by gohoyer** [comment]
Implemented on [Release 1.1.1](https://github.com/gohoyer/Alfred-Cheat.sh/releases/tag/v1.1.1)

**Comment by gohoyer** [comment]
> just a minor thing since it seems you also develop small Alfred workflow:
> 
> Change this to `external script` to save the file seperately from the `info.plist`. That enables other's to easier see the underlying code and make PRs (e.g. I could have made a PR for this issue to save you some time 🙂 )
> 
> <img alt="Screenshot 2022-02-14 16 39 22" width="333" src="https://user-images.githubusercontent.com/73286100/153895760-e57b16f3-d064-4a0b-826b-99bd8b9112ac.png">

Done!
